gems/pending/spec/postgres_ha_admin/failover_monitor_spec.rb

```diff
@@ -15,9 +15,9 @@
   end
 
   let(:failover_monitor) do
-    expect(PostgresHaAdmin::DatabaseYml).to receive(:new).and_return(db_yml)
-    expect(PostgresHaAdmin::FailoverDatabases).to receive(:new).and_return(failover_db)
-    failover_instance = described_class.new('', '', @logger_file.path, 'test')
+    allow(PostgresHaAdmin::DatabaseYml).to receive(:new).and_return(db_yml)
+    allow(PostgresHaAdmin::FailoverDatabases).to receive(:new).and_return(failover_db)
```

Comment:
This code will only run when you call `failover_monitor` in a spec, so if you need to use a different initialization you probably shouldn't use `failover_monitor`.